src/levanter/store/cache.py

```diff
@@ -473,7 +473,11 @@ def _monitor_metrics(self):
 class CacheLedger:
     # NB: unlike the old cache, the mere existence of a ledger doesn't mean the cache is finished
     total_num_rows: int
-    shard_rows: Dict[str, int]
+    """Number of outputted rows in the cache"""
+    shard_rows_in: Dict[str, int]
```

Comment:
sorry, can we rename this one back to just shard_rows (leave comment) so that we don't invalidate all other caches

Reply:
Ah gotcha yeah... Do I need to do some other logic to make `shard_rows_out` an optional key during deserialization?

Reply:
i'm pretty sure if you give it a default value it will be fine. I'll check it against a cache before merging